Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -58,6 +68,9 @@ private set
             }
         }
 
+
+        [SerializeField] public UnityEvent<DateTime> OnCurrentDateTimeChanged = new();
```

Comment:
We hadden toch besproken dat deze NonSerialized moet zijn? Of heb je die changes nog niet gepushed?